devices/Ssd13xx/Ssd13xx.cs

```diff
@@ -311,6 +349,18 @@ public void ClearDirectAligned(int x, int y, int width, int height)
             }
         }
 
+        /// <summary>
+        /// In-place swap of a and b values without the use of a temp variable.
+        /// </summary>
+        /// <param name="a">1st variable to be swapped.</param>
+        /// <param name="b">2nd variable to be swapped.</param>
+        private void Ssd1306Swap(ref int a, ref int b)
```

Comment:
naming wise, why Ssd1306? Is the swap specific? If not, then just call Swap. and if specific, how to swap for the other screens?

Reply:
Yeah, I was borrowing code from the AdaFruit C++ driver where it's written this way.  I'll change it to just Swap.